.github/BOTMETA.yml

```diff
@@ -43,10 +43,14 @@ files:
   changelogs/fragments/:
     support: community
   $modules/cloud/amazon/:
-    ignored: erydo seiffert simplesteph nadirollo
+    ignored: erydo seiffert simplesteph nadirollo tedder
   $modules/cloud/amazon/aws_api_gateway.py: willthames
-  $modules/cloud/amazon/aws_kms.py: willthames
-  $modules/cloud/amazon/cloudformation.py: ryansb tedder
+  $modules/cloud/amazon/aws_kms.py:
+    maintainers: willthames
+    ignored: tedder
```

Comment:
This should not be needed as you are already ignored for the complete directory.